### DISCUSSION TITLE
Allow the use of ENV variables in Flow Operations

### DISCUSSION THREAD

**Discussion by metalmarco** [directus/directus#14054] [Feature Requests]
When setting up a Webhook operation in a Flow, it should be possibile to use an ENV variable to populate part of the URL field.
Currently it's  not possibile, for example, to differentiate URLs endpoints based of deploy environments (Staging, Pre-Production and Production)

<img width="909" alt="Schermata 2022-06-23 alle 10 50 01" src="https://user-images.githubusercontent.com/7231582/175258227-7d529d4c-ce1c-451a-8ca9-461b37ea4911.png">

### To Reproduce

- Set up a new Flow with a Webhook operation
- URL doesn't allow you to specify an ENV variable to be used

**Comment by azrikahar** [comment]
Should be similar to #13948. Although that is focused on Flows, the underlying roots of implementation _should_ be interoperable between Webhooks/Flows I think (or potentially Flows focused going forward).

**Reply by benhaynes** [reply]
The dedicated webhooks feature will be deprecated in the upcoming major release, in favor of the more flexible Flows feature. So we should focus on improving flows, not webhooks.

**Reply by metalmarco** [reply]
Hi Ben, thanks for the answer.
The same request applies to Flow (which we are currently using instead of webhooks), so i've edit the issue accordingly.

**Comment by NeoKactus** [comment]
I have the same use case. I suggested a potential workaround here. https://github.com/directus/directus/discussions/13948#discussioncomment-3019245

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
